modules/sdk-coin-algo/test/unit/lib/transactionBuilder/assetTransferBuilder.ts

```diff
@@ -2,9 +2,9 @@ import { coins } from '@bitgo/statics';
 import assert from 'assert';
 import should from 'should';
 import { test } from 'mocha';
```

Comment:
We might need to add mocha into our devDependencies I think.

Reply:
actually `mocha` needs to only be in the root package. We ran into issues with it being included in multiple places when refactoring TestBitgo.

Reply:
I see. Then everything looks good to me! Approving it right now.